### PR TITLE
Rename eShopLite to TestShop

### PR DESCRIPTION
**Pull request by eerhardt**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/2095)

### CODE REVIEW

**Comment by danmoseley** [comment]
Is anyone going to read AspireShop and think it might represent something they should look at and copy from? That's why I was thinking of a more random name.
But whatever..

**Comment by JamesNK** [comment]
✨ TestShop ✨

**Comment by eerhardt** [comment]
✨ BikeShedShop✨

File: docs/contributing.md [hunk 1]
```diff
@@ -9,20 +9,20 @@ See [machine-requirements.md](machine-requirements.md).
...
-1. Set the Startup Project to be the `AppHost` project (it's under `\samples\eShopLite`). Make sure the launch profile is set to "http".
+1. Set the Startup Project to be the `AppHost` project (it's under `\playground\TestShop`). Make sure the launch profile is set to "http".
 1. <kbd>F5</kbd> to debug, or <kbd>Ctrl+F5</kbd> to launch without debugging.
 
 Otherwise:
 ```shell
-dotnet restore samples/eShopLite/AppHost/AppHost.csproj
-dotnet run --project samples/eShopLite/AppHost/AppHost.csproj
+dotnet restore playground/TestShop/TestShop/AppHost.csproj
+dotnet run --project playground/TestShop/TestShop/AppHost.csproj
```
Comment (eerhardt) [inline docs/contributing.md:25]:
```suggestion
dotnet restore playground/TestShop/AppHost.csproj
dotnet run --project playground/TestShop/AppHost.csproj
```

File: docs/contributing.md [hunk 2]
```diff
@@ -9,20 +9,20 @@ See [machine-requirements.md](machine-requirements.md).
...
-1. Set the Startup Project to be the `AppHost` project (it's under `\samples\eShopLite`). Make sure the launch profile is set to "http".
+1. Set the Startup Project to be the `AppHost` project (it's under `\playground\TestShop`). Make sure the launch profile is set to "http".
 1. <kbd>F5</kbd> to debug, or <kbd>Ctrl+F5</kbd> to launch without debugging.
 
 Otherwise:
 ```shell
-dotnet restore samples/eShopLite/AppHost/AppHost.csproj
-dotnet run --project samples/eShopLite/AppHost/AppHost.csproj
+dotnet restore playground/TestShop/AppHost.csproj
+dotnet run --project playground/TestShop/AppHost.csproj
```
Comment (eerhardt) [inline docs/contributing.md:25]:
```suggestion
dotnet restore playground/TestShop/AppHost/AppHost.csproj
dotnet run --project playground/TestShop/AppHost/AppHost.csproj
```

**Comment by eerhardt** [comment]
Any thoughts here? I went with `TestShop`, thanks to @JamesNK suggestion.

**Review by JamesNK — APPROVED** [review]

**Comment by danmoseley** [comment]
other than Violet 😄  ? testshop seems fine to me

**Pull request merged**